docker/cli
# [18.06] vndr of buildkit to 98f1604

**- What I did**

update buildkit vndr to 98f1604 and the associated vndrs of the vendor.conf of buildkit

for reference, comparison of upstream changes: https://github.com/moby/buildkit/compare/9acf51e49185b348608e0096b2903dd72907adcb...98f1604134f945d48538ffca0e18662337b4a850

also updated containerd and units to match the git commit hash of the buildkit `vendor.conf`

**- How I did it**

```
$ vi vendor.conf
$ make -f docker.Makefile shell
$ vndr github.com/moby/buildkit
```

**- How to verify it**
```
$ make -f docker.Makefile vendor
...
2018/08/14 06:28:44 Success
2018/08/14 06:28:44 Running time: 1m31.138362914s
scripts/validate/check-git-diff vendor
vendor is correct
```
**- Description for the changelog**


* Update buildkit to 98f1604

**- A picture of a cute animal (not mandatory but encouraged)**

🐍
